packages/ui-components/style/base.css

```diff
@@ -51,12 +51,13 @@ strong {
   font-weight: unset;
 }
 
-/* Override Blueprint's _typography.scss styles */
-a {
+/* Override Blueprint's _typography.scss styles outside of dialog boxes */
+a:not(.jp-VisibleLink) {
```

Comment:
Could we just use `.jp-Dialog-content` here rather than introducing a new class (which needs to be manually added)? I am sure there are links in dialogs of extensions (and possibly in other dialogs in core) which suffer from lack of distinction.

Reply:
Reverted the new `jp-VisibleLink` class and added two new CSS rules to override the `a` and `a:hover` rules.